keel-web/src/test/resources/examples/titus-cluster-with-test-container.yml

```diff
@@ -0,0 +1,26 @@
+---
+application: acme
+artifacts:
+  - name: rocketskates
+    type: docker
+    reference: rocketskates
+environments:
+  - name: test
+    locations:
+      account: test
+      regions:
+        - name: us-east-1
+    resources:
+      - kind: ec2/cluster@v1.1
```

Comment:
```suggestion
      - kind: titus/cluster@v1
```

Reply:
I don't understand how the tests are passing with the delivery config in this state... 🤔

Reply:
> I don't understand how the tests are passing with the delivery config in this state... 🤔

Do you mean, because the artifact type doesn't match the resource type?

I'd like to build on this to write a more extensive set of tests against delivery configs to catch more invalid configsthat we aren't checking for yet.

Reply:
No, I mean because that `container` field you have there is not part of `ClusterSpec` (EC2), so I don't understand how that would parse correctly.